### PR TITLE
[1LP][RFR] Automating test_superadmin_child_tenant_delete_parent_catalog

### PR DESCRIPTION
**Pull request by mnadeem92**
Automating Multi-Tenancy Test Case RHCF3-35087: test_superadmin_child_tenant_delete_parent_catalog

{{ pytest: cfme/tests/configure/test_access_control.py::test_superadmin_child_tenant_delete_parent_catalog --use-template-cache }}

### CODE REVIEW

File: cfme/tests/configure/test_access_control.py [hunk 1]
```diff
@@ -1678,7 +1677,34 @@ def test_superadmin_child_tenant_delete_parent_catalog():
...
+    catalog_name = fauxfactory.gen_alphanumeric()
+    cat = appliance.collections.catalogs.create(name=catalog_name, description='my catalog')
+    new_tenant = tenant_collection.create(
+        name="tenant{}".format(fauxfactory.gen_alpha(4)),
+        description=fauxfactory.gen_alphanumeric(16),
+        parent=root_tenant)
+
+    group = group_collection.create(description='grp{}'.format(fauxfactory.gen_alphanumeric()),
+                                    role="EvmRole-super_administrator",
+                                    tenant=root_tenant.name + "/" + new_tenant.name)
```
Comment (digitronik) [inline cfme/tests/configure/test_access_control.py]: Required:
Please use `.format` for string formatting.

File: cfme/tests/configure/test_access_control.py [hunk 2]
```diff
@@ -1678,7 +1677,34 @@ def test_superadmin_child_tenant_delete_parent_catalog():
...
+    @request.addfinalizer
+    def _delete_user_group_tenant():
+        for item in [user, group, new_tenant]:
+            if item.exists:
+                item.delete()
+
+    assert user.exists
+    try:
+        with user:
+            navigate_to(appliance.server, 'LoggedIn')
```
Comment (digitronik) [inline cfme/tests/configure/test_access_control.py]: I don't think it's required. please try cat.delete() directly

File: cfme/tests/configure/test_access_control.py (hunk 1 above)
Comment (mnadeem92) [reply cfme/tests/configure/test_access_control.py]: Done

File: cfme/tests/configure/test_access_control.py (hunk 2 above)
Comment (mnadeem92) [reply cfme/tests/configure/test_access_control.py]: Agree

File: cfme/tests/configure/test_access_control.py [hunk 3]
```diff
@@ -1678,7 +1677,33 @@ def test_superadmin_child_tenant_delete_parent_catalog():
...
+        for item in [user, group, new_tenant]:
+            if item.exists:
+                item.delete()
+
+    try:
+        with user:
+            navigate_to(appliance.server, 'LoggedIn')
+            cat.delete()
+            assert not cat.exists
+    finally:
```
Comment (mshriver) [inline cfme/tests/configure/test_access_control.py]: :ok_hand:

**Pull request merged**